core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -154,6 +154,9 @@ func NewEthBroadcaster(
 		isStarted:        false,
 		autoSyncNonce:    autoSyncNonce,
 	}
+
+	b.processUnstartedEthTxsImpl = b.processUnstartedEthTxs
```

Comment:
Oh ok, so you moved this assignment here.
Yes, it work, as long as everyone that's not unit-test, instantiates EthBroadcaster via this method.
I think we can safely assume there won't be other callers outside of this function.